src/types.rs

```diff
@@ -31,15 +31,27 @@ pub struct BalanceResponse {
     #[doc = " Total computed balance"]
     pub value: u64,
 }
+#[doc = " The value of the Crossover and the blinder"]
+#[derive(Clone, PartialEq, Debug, Deserialize, Serialize)]
+pub struct CrossoverType {
```

Comment:
I would suggest changing this for something more descriptive. Adding `Type` doesn't convey what it is. To help, here's some questions:

- What is this doing?
- What is it meant to do?

Reply:
It's the only way to transmit the crossover to the wasm side, that's why I put it to be "CrossoverType" since other types in the json schema have the suffix "Type". It has the blinder and the value that was used to construct the crossover and the phoenix_crossover itself